randovania/gui/tracker_window.py

```diff
@@ -492,17 +492,10 @@ def setup_elevators(self):
         region_list = self.game_description.region_list
         nodes_by_region: dict[str, list[DockNode]] = collections.defaultdict(list)
 
-        areas_to_not_change = {
-            "Sky Temple Gateway",
-            "Sky Temple Energy Controller",
-            "Aerie Transport Station",
-            "Aerie",
-        }
```

Comment:
`tracker_window.py` my favorite file...this can be removed because it was only relevant for the teleporters and not the elevators?

Reply:
this can be removed because a few **years** ago these were made shuffleable and should've been removed back then